src/components/homepage/homepage.component.html

```diff
@@ -1 +1,41 @@
+<div class="hero-banner__container">
+	<div class="hero-banner__text">
+		<div class="hero-banner__text-heading">Be The Reason</div>
+		<div class="hero-banner__text-heading">Someone Smiles Today</div>
+		<div class="hero-banner__text-description">
+			Our platform bridges the gap between those with an abundance and those in
+			need, fostering a community of kindness and generosity. Together, we turn
+			surplus into sustenance, transforming every meal into a beacon of hope and
+			compassion.
+		</div>
+		<div class="hero-banner__text-description">
+			Join us in spreading love, one plate at a time !
 
+		</div>
+		<div class="hero-banner__text-buttons">
+			<button type="button" class="hero-banner__text-buttons-donate btn">
+				Donate
```

Comment:
Rather than "Donate" it should be something like "Donate Food" . Just check once